### PR TITLE
Add toggle to disable / enable mobile touch

### PR DESCRIPTION
**Pull request by mozky**
Hey @rcaferati, I added this small PR that adds a new prop to the component called `touch` with a default value of `true`. This boolean prop will set the status of the mobile swipe effect that advanced to the next slide.

Hit me up if you require any changes

Solves issue #100

### CODE REVIEW

File: index.d.ts [hunk 1]
```diff
@@ -55,6 +55,7 @@ declare module 'react-awesome-slider' {
     startupScreen?: object;
     style?: object;
     transitionDelay?: number;
+    touch?: boolean;
```
Comment (rcaferati) [inline index.d.ts]: Change the `touch` to `mobileTouch` as it's just available on mobile.

File: src/core/index.js [hunk 2]
```diff
@@ -961,8 +943,38 @@ export default class AwesomeSlider extends React.Component {
             }}
             className={this.classNames.container}
           >
-            {this.renderBox('A')}
-            {this.renderBox('B')}
+            <div
```
Comment (rcaferati) [inline src/core/index.js]: Any specific reason for removing the `renderBox` method and duplicating the code? If not just roll it back.

Comment (mozky) [reply src/core/index.js]: I removed `renderBox` to have direct access to the new prop `mobileTouch`, I can bring back `renderBox` and pass `mobileTouch` as a second argument to the function if you prefer.

Comment (rcaferati) [reply src/core/index.js]: Yes, that'd be great

**Comment by mozky** [comment]
@rcaferati I believe it's ready to go.

**Comment by rcaferati** [comment]
Awesome, thank you! o/

**Pull request merged**